swagger/public_v1/component_schemas/CourseAttributes.yml

```diff
@@ -220,10 +238,10 @@ properties:
     type: string
     description: "Unique code for the provider of this course."
     maxLength: 3
-    minLength: 3
+    minLength: 2
```

Comment:
It should be 3.

For reference see https://github.com/DFE-Digital/teacher-training-api/pull/1504/commits/da089905ddf60456dc51727ca456865a0d5b3b69

Reply:
I've changed this back to 3 now